run

```diff
@@ -1,53 +1,118 @@
 #!/usr/bin/env python3
 
-import getopt
-import os
-import sys
+import click
 
 from webtranslate import main
 
-
-def run():
+CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--server-mode",
+    help="Mode of server",
+    type=click.Choice(["development", "production"], case_sensitive=False),
+    default="development",
+)
+@click.option("--server-host", help="Host to bind the server to", default="127.0.0.1")
+@click.option("--server-port", help="Port to bind the server to", default=8000)
+@click.option(
+    "--authentication",
+    help="Authentication protocol to use",
+    type=click.Choice(["development", "github"], case_sensitive=False),
+    default="development",
+)
+@click.option("--stable-languages", help="Folder with stable languages", default="stable_languages")
+@click.option("--unstable-languages", help="Folder with unstable languages", default="unstable_languages")
+@click.option("--project-root", help="Root folder for data storage", required=True)
+@click.option("--project-cache", help="I have no idea", default=1)
+@click.option(
+    "--project-types",
+    help="I have no idea",
+    multiple=True,
+    type=click.Choice(["newgrf", "game-script", "openttd"], case_sensitive=False),
+    default=["newgrf", "game-script"],
+)
+@click.option(
+    "--storage-format",
+    help="No clue",
+    type=click.Choice(["one-file", "split-languages"], case_sensitive=False),
+    default="one-file",
+)
+@click.option(
+    "--data-format",
+    help="Format to store project data in",
+    type=click.Choice(["xml", "json"], case_sensitive=False),
+    default="xml",
+)
+@click.option("--language-file-size", help="really, wuth?", default=100000)
+@click.option("--num-backup-files", help="How many backup files for project data to keep", default=5)
+@click.option("--max-num-changes", help="Sometimes important I am sure", default=5)
+@click.option("--min-num-changes", help="another very important flag", default=2)
```

Comment:
Yeah, I don't know either... I know it must be >= 2 or the git scripts fail.
Maybe hardcode to 2 here?